services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/logging/DisclosureLogService.kt

```diff
@@ -47,6 +48,17 @@ class DisclosureLogService(private val auditLogService: AuditLogService) {
         saveDisclosureLogs(ALLU_AUDIT_LOG_USERID, UserRole.SERVICE, entries)
     }
 
+    /**
+     * Save disclosure logs for when a user downloads a decision.We don't know what information is
```

Comment:
a decision.We don't -> a decision. We don't

Reply:
Fixed.